source/testing/index.md

```diff
@@ -5,39 +5,73 @@ This application would likely include models such as `user` and `post`. It would
 also include interactions such as _login_ and _create post_. Let's finally
 assume that you would like to have [automated tests] in place for your application.
 
-There are two different classifications of tests that you will need:
-**Acceptance** and **Unit**.
+There are three different classifications of tests that you will need:
+**Acceptance**, **Unit**, and **Integration**.
 
 ### Acceptance Tests
 
-Acceptance tests are used to test user interaction and application flow. With
-the example scenario above, some acceptance tests you might write are:
+Acceptance tests are used to test the application end to end.  The tests interact
```

Comment:
It was meant to convey that you typically don't isolate parts of the application during acceptance tests.  They are meant to test the whole system.  I can back down from that though, because I have heard of folks running acceptance tests with a mirage back end before the apis are ready.